packages/orbit-components/src/NavigationBar/index.tsx

```diff
@@ -74,7 +74,7 @@ const NavigationBar = ({
       )}
       aria-label={ariaLabel}
     >
-      <div className="me-200 block w-full">{children}</div>
+      <div className={cx("block w-full", onMenuOpen && "me-200")}>{children}</div>
```

Comment:
I added this condition for the margin as it doesn't make sense to me for it to be there if there is no menu link rendered but I might be missing some information why it's there.

Reply:
We have confirmation from the designers to remove the margin.

[Internal Slack thread](https://skypicker.slack.com/archives/C7T7QG7M5/p1736861994687889?thread_ts=1736161053.827579&cid=C7T7QG7M5)